extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -98,8 +99,18 @@ define('ext.wikia.adEngine.lookup.services', [
 		return slotPrices;
 	}
 
+	function storeRealSlotPrices(slotName) {
+		realSlotPrices[slotName] = getCurrentSlotPrices(slotName);
+	}
+
+	function getRealSlotPrices(slotName) {
```

Comment:
What do you think of renaming the methods with "real" world? :) For example: `s/getRealSlotPrices/getDfpSlotPrices/`?